core/design/src/main/kotlin/edu/stanford/spezi/core/design/views/views/layout/DescriptionGridRow.kt

```diff
@@ -0,0 +1,75 @@
+package edu.stanford.spezi.core.design.views.views.layout
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.HorizontalDivider
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import edu.stanford.spezi.core.design.theme.ThemePreviews
+
+@Composable
+fun DescriptionGridRow(
+    description: @Composable () -> Unit,
```

Comment:
It would be great to type indicate that description and content are rendered in a box, will be helpful for callers to use box specific layout config if needed. I propose the following (we can use 0.5f weight instead - it is more readable imo to indicate that both sides are weighted equally, half of available width)

```kotlin
@Composable
fun DescriptionGridRow(
    modifier: Modifier = Modifier,
    description: @Composable BoxScope.() -> Unit,
    content: @Composable BoxScope.() -> Unit,
) {
    Row(
        modifier = modifier
            .fillMaxWidth()
            .padding(Spacings.small),
        horizontalArrangement = Arrangement.spacedBy(8.dp),
        verticalAlignment = Alignment.CenterVertically
    ) {
        Box(
            modifier = Modifier
                .alignByBaseline()
                .weight(0.5f),
            content = description,
        )

        Box(
            modifier = Modifier
                .alignByBaseline()
                .fillMaxWidth()
                .weight(0.5f),
            content = content,
        )
    }
}
```

Reply:
I removed these weight modifiers entirely